v1/formats/nswpdc7and8.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name>NSW Premier's Debating Challenge (7 and 8)</name>
+  <short-name>NSWPDC7and8</short-name>
+  <version>1</version>
+  <info>
+    <region>NSW, Australia</region>
+    <level>Primary School</level>
+    <used-at>NSW Premier's Debating Challenge (for Years 7 and 8)</used-at>
+    <description>3 speakers + 4th vs 3 speakers + 4th, 6 minute speeches</description>
```

Comment:
Being a bit picky but I'd use something similar to the Australian Easters description, like "3 vs 3, 6-minute speeches, no replies, no POIs". If you want to acknowledge the fourth prep teammate we could, but the World Schools format doesn't do so and the purpose is mostly to describe the timings/format rather than prep rules.